src/server/player_sao.h

```diff
@@ -184,6 +184,8 @@ class PlayerSAO : public UnitSAO
 	void unlinkPlayerSessionAndSave();
 	std::string generateUpdatePhysicsOverrideCommand() const;
 
+	ItemStack getItemFromIndex(std::string, int i) const;
```

Comment:
unused

Reply:
Wha... I don't even remember adding this, lol.

Reply:
Done